assets/css/navbar-responsive.css

```
@@ -0,0 +1,451 @@
+  }
+
+  .site-header #nav.navbar.is-fresh .navbar-menu .navbar-mobile-social a:hover,
+  #navbar-clone.navbar.is-fresh .navbar-menu .navbar-mobile-social a:hover {
+    color: var(--openms-orange);
+  }
+
+  .site-header #nav.navbar.is-fresh .navbar-menu .navbar-mobile-social svg.icon,
+  #navbar-clone.navbar.is-fresh .navbar-menu .navbar-mobile-social svg.icon {
+    width: 1.5rem;
+    height: 1.5rem;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)